ansible/README.md

```diff
@@ -43,7 +43,8 @@ These playbooks are available to you:
   - **jenkins/linter.yml**: Sets up the code linters (flavour of a worker).
 
   - **create-webhost.yml**: Configures the server(s) that host nodejs.org,
-                            iojs.org and dist.libuv.org among other things.
+                            iojs.org and dist.libuv.org among other things
+                            (only works on Ubuntu boxes).
```

Comment:
Not perfectly happy with my last response. Let me elaborate for posterity!

So, as I wrote this; I added two hosts to the playbook; both of which have/are "ubuntu". I can understand that if someone is in the process of helping out by testing different playbooks in different environments, running into "hey this broke on !ubuntu" would point to adding similar remarks.